config/rca.conf

```diff
@@ -65,6 +65,14 @@
     "shard-request-cache-rca": {
       "shard-request-cache-threshold" : 0.9
     },
+    "search-back-pressure-rca": {
+       "max-heap-usage-increase": 80,
+       "max-shard-heap-cancellation-percentage": 5,
+       "max-task-heap-cancellation-percentage": 5,
+       "max-heap-usage-decrease":  90,
+       "min-shard-heap-cancellation-percentage": 3,
```

Comment:
Why just 3? Should we go even lower to 2 or maybe 1?

Reply:
There is no harm in making it 1% but I think it would be highly unlikely that the task cancellations due to heap are below and still heap is contended (when there are lots of low mem tasks are present). But I think it is a good point there is no harm in making it 1 as well.